src/components/stepforms/widgets/FilterSimpleCondition.vue

```diff
@@ -2,21 +2,18 @@
   <div class="filter-form-simple-condition__container">
     <div class="filter-form-simple-condition-column-input">
       <AutocompleteWidget
-        :id="`${dataPath.replace(/[^a-zA-Z0-9]/g, '')}-columnInput`"
-        v-model="value.column"
+        id="columnInput"
```

Comment:
This id will not be unique... That why I suggested: ` :id="`${dataPath.replace(/[^a-zA-Z0-9]/g, '')}-columnInput`"` Why did you remove it ? :)